go/cmd/dolt/cli/help.go

```diff
@@ -101,6 +160,52 @@ const (
 
 var bold = color.New(color.Bold)
 
+func handleAngleBrackets(str string) string {
+	var res string
+	var start int
+	var end int
+
+	curr := str
+
+	for {
```

Comment:
Why don't you just strings.replaceAll? This would require doing a first pass to change &lt;b&gt; and other supported tags into markdown equivalents.

Reply:
yeah, thats much cleaner.